exporter/awsemfexporter/emf_exporter.go

```diff
@@ -29,9 +29,9 @@ const (
 	outputDestinationCloudWatch = "cloudwatch"
 	outputDestinationStdout     = "stdout"
 
-	// Pulse EMF config
-	pulseMetricNamespace    = "AWS/APM"
-	pulseLogGroupNamePrefix = "/aws/apm/"
+	// AppSignals EMF config
+	appSignalsMetricNamespace    = "AppSignals"
```

Comment:
Are we omitting the AWS/ prefix? I think we should still do this as it is reserved

Reply:
Spoke offline, this follows same pattern as container insights